l2tdevtools/dependencies.py

```diff
@@ -504,6 +504,40 @@ def GetInstallRequires(self):
 
     return sorted(install_requires)
 
+  def GetPipenvPackages(self):
```

Comment:
Please remove all references to pipenv from this - just mention the pipfile.

Reply:
Why?

Reply:
See above - this file is generating a pipfile, which is a file format that might be read by multiple tools. The packages in the file are not specific to pipenv.